docs/REST API/rest-api.md

```diff
@@ -8805,6 +8806,42 @@ echo "hello"
 }
 ```
 
+### Modify script
+
+Modifies an existing script.
+
+`PATCH /api/v1/fleet/scripts/:id`
+
+##### Request body
+
+```json
+{
+  "script_content": "#!/bin/sh\\n\\n#!/usr/bin/env bash\\n\\nsudo systemsetup -settimezone Pacific/Ponape"
+}
+```
+
+#### Parameters
+
+| Name            | Type    | In   | Description                                           |
+| ----            | ------- | ---- | --------------------------------------------          |
+| id              | integer | path | **Required**. The ID of the script to modify. |
+| script_contents | string  | body | The contents of the script to run. Only one of either `script_id` or `script_contents` can be included in the request; omit this parameter if using `script_id`. |
```

Comment:
Let's make this required since it's the only parameter other than `id`. Otherwise, you're not updating anything; may as well fail loudly if you try to update a script without including the script contents. (If we add other parameters to this endpoint in the future, we should make this parameter optional at that time.)

Also, we can remove the note about only specifying one or the other (that's just for running a script).
```suggestion
| script_contents | string  | body | **Required**. The contents of the script to be updated. |
```